client/components/tinymce/plugins/mentions/style.scss

```diff
@@ -0,0 +1,66 @@
+.suggestions {
+	width: 40%;
+
+	img {
+		border-radius: 50%;
+		width: 24px;
+		height: 24px;
+		float: left;
+		margin: 0 10px 0 0;
+	}
+
+	strong {
+		font-weight: 400;
+		background: rgba( $blue-light, 0.25 );
+	}
+
+	.username {
+		float: left;
+		display: inline-block;
+		font-weight: 400;
+		padding: 10px 0;
+		margin: -10px 0 0 0;
+	}
+
+	small {
+		font-size: 11px;
+		font-weight: 400;
+		float: right;
+		display: inline-block;
+	}
+}
+
+.rtl {
```

Comment:
@yoavf Could you let me know which .scss file is the most appropriate place to put the ltr rule? Thx!